react-native/react/store/configure-store.native.desktop.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+/* @flow */
+
+import { createStore, compose } from 'redux'
+import { devTools } from 'redux-devtools'
+
+export default function configureStoreNative (rootReducer, initialState, createStoreWithMiddleware) {
+  const nativeCreateStoreWithMiddleware = compose(
+    createStoreWithMiddleware,
+    devTools()
+  )(createStore)
+
+  return nativeCreateStoreWithMiddleware(rootReducer, initialState)
+}
```

Comment:
You still could if you wanted to, just by returning a function that takes `rootReducer` and `initialState`, but these versions would get much simpler.

`configure-store.native.desktop.js`:

```js
export default f => compose(f, devTools())(createStore)
```

`configure-store.native.mobile.js`:

```js
export default f => f(createStore)
```

A hypothetical one that needs to deal with `rootReducer` and `initialState`:

```js
export default f => (rootReducer, initialState) => { … }
```